doc/lsp.txt

```diff
@@ -320,6 +320,44 @@ Additionally the following configurations can be made:
 	omnicompl	(Optional) a boolean value that enables (true)
 			or disables (false) omni-completion for this file
 			types. By default this is set to "v:true".
+						*lsp-cfg-processDiagHandler*
+	processDiagHandler
+			(Optional) A |Funcref| or |lambda| that takes a list of
+			language server diagnostics and returns a new list of
+			filtered, or otherwise changed diagnostics.  Can be used
+			to remove unwanted diagnostics, prefix the diagnostics
+			text, etc.  The following example will remove all but
+			errors and warnings: >
+
+		vim9script
+		g:LspAddServer([{
+			filetype: ['javascript', 'typescript'],
+			path: '/usr/local/bin/typescript-language-server',
+			args: ['--stdio'],
+			processDiagHandler: (diags: list<dict<any>>) => {
+				# Only include errors and warnings
+				return diags->filter((diag, ix) => {
+					return diag.severity <= 2
```

Comment:
technically maybe ok, but  i doubt is it really a right approach? 
i am feeling some kinds of risk which maybe used to manipulate sth if had such things... :rofl: 
// user maybe just copied cfg from somewhere/someone, but didnot understand what's that for, 
// a.k.a it maybe dangerous.